bool_slice.go

```diff
@@ -0,0 +1,125 @@
+package pflag
+
+import (
+	"strconv"
+	"strings"
+)
+
+// -- boolSlice Value
+type boolSliceValue struct {
+	value   *[]bool
+	changed bool
+}
+
+func newBoolSliceValue(val []bool, p *[]bool) *boolSliceValue {
+	bsv := new(boolSliceValue)
+	bsv.value = p
+	*bsv.value = val
+	return bsv
+}
+
+func (s *boolSliceValue) Set(val string) error {
+	ss := strings.Split(val, ",")
```

Comment:
This is what we do in intSlice, isn't it? Trying to decide if I like it. I know that moving to using `readAsCSV` and `writeAsCSV` made things much more predictable. I'm trying to decide what I think should be the result of:
```
--boolslice="true,false",false
```
This cod would fail because `"true` was invalid. CSV code would fail because `true,false` was invalid.

Do we care? I personally would use CSV parsing throughout, but I don't have a good argument why